java/compaction/compaction-job-creation/src/main/java/sleeper/compaction/job/creation/CreateCompactionJobs.java

```diff
@@ -114,38 +117,57 @@ public enum Mode {
     }
 
     public void createJobs(TableProperties table) throws StateStoreException, IOException, ObjectFactoryException {
-        LOGGER.info("Performing pre-splits on files in {}", table.getStatus());
+        LOGGER.info("Creating compaction jobs for table {}", table.getStatus());
         StateStore stateStore = stateStoreProvider.getStateStore(table);
+        LOGGER.info("Performing pre-splits on files in table {}", table.getStatus());
+        Instant preSplitStartTime = Instant.now();
         SplitFileReferences.from(stateStore).split();
-        createJobsForTable(table, stateStore);
+        LOGGER.info("Pre-split files in partitions, took {}", LoggedDuration.withShortOutput(preSplitStartTime, Instant.now()));
+        Instant finishTime = createJobsForTable(table, stateStore);
+        LOGGER.info("Overall, creating compaction jobs took {}", LoggedDuration.withShortOutput(preSplitStartTime, finishTime));
```

Comment:
This seems a bit misleading, it doesn't sound like this would include pre-splits.

Reply:
Thanks, ive updated this log message